packages/elements/package.json

```diff
@@ -43,7 +43,7 @@
   "scripts": {
     "build": "stencil build --docs --legacy && node scripts/generateWrappedIcons.js",
     "clean": "shx rm -rf dist",
-    "dev": "stencil build --docs --watch --legacy",
+    "dev": "NODE_ENV=development stencil build --docs --watch --legacy",
```

Comment:
Nice I didn't know about the sourceMap Feature. If we want to make sure that env settings work also on Windows environment we may use one of the following: 

* https://github.com/kentcdodds/cross-env
* https://github.com/toddbluhm/env-cmd

Reply:
You are right. I'll change it to use `cross-env` 👍🏼